src/pages/register-a-beacon/aircraft-communications.tsx

```diff
@@ -99,33 +89,15 @@ const AircraftCommunications: FunctionComponent<FormPageProps> = ({
   const pageHeading = "How can we communicate with you, when on this aircraft?";
 
   return (
-    <Layout
-      navigation={
-        <BackButtonRouterIndexes href="/register-a-beacon/about-the-aircraft" />
-      }
-      title={pageHeading}
-      pageHasErrors={form.hasErrors}
+    <BeaconsForm
+      previousPageUrl="/register-a-beacon/about-the-aircraft"
+      pageHeading={pageHeading}
       showCookieBanner={showCookieBanner}
+      formErrors={form.errorSummary}
+      insetText="This will be critical for Search and Rescue in an emergency."
     >
-      <Grid
-        mainContent={
-          <>
-            <PageHeading>{pageHeading}</PageHeading>
-            <FormErrorSummary formErrors={form.errorSummary} />
-            <GovUKBody>
-              This will be critical for Search and Rescue in an emergency.
-            </GovUKBody>
-
-            <Form>
-              <TypesOfCommunication form={form} />
-
-              <Button buttonText="Continue" />
-            </Form>
-            <IfYouNeedHelp />
-          </>
-        }
-      />
-    </Layout>
+      <TypesOfCommunication form={form} />
+    </BeaconsForm>
   );
```

Comment:
Use of `<BeaconsForm>` component necessary to handle uses in the `<HiddenFormMetadata>` component.